### DISCUSSION TITLE
.NET Framework 4.7.2: missing assembly "System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"

### DISCUSSION THREAD

**Discussion by svenlohberg** [Shane32/QRCoder#400] [General]
I am building an application for .NET Framework 4.7.2. When using QRCoder 1.4.3 I get an exception, that the following assembly is missing: "System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a".

Switching one version back to QRCoder 1.4.2 did solve the problem.

**Comment by gustavobosch** [comment]
Set .Net Framework 8.0 as target framework instead of 4.7.2. It worked for me when i made that change

**Reply by Shane32** [reply]
Do you have problems with the current version of QRCoder under .NET Framework?
